elife/certificates.sls

```diff
@@ -22,26 +22,28 @@ web-certificates-dir:
             - web-certificates-group
 
 {% if salt['elife.cfg']('cfn.outputs.DomainName') %}
+{% set app = pillar.elife.certificates.app or 'elife' %}
```

Comment:
do we actually need the default `or 'elife'`? `pillar/elife.sls` is always going to provide the `elife` value

Reply:
you're right, it is redundant. I felt better safe than sorry should there be a gap between builder-base being merged in before the builder-private change